.github/workflows/debci.yml

```diff
@@ -50,6 +51,8 @@ jobs:
           cp -r netplan.io-*/debian .
           rm -r debian/patches/  # clear any distro patches
           sed -i 's|rm debian/tmp/lib/netplan/generate|# DELETED|' debian/rules
+          sed -i 's|bytecompile=-1|bytecompile=-1 -Dtesting=false|' debian/rules
```

Comment:
question: is this still needed? In the autopkgtest we already set use `--env=DEB_BUILD_OPTIONS=nocheck` and I saw that you already updated netplan.io on Debian. Well, I suppose it's here because the package wasn't published yet...

Reply:
Yes. The new `autopkgtest` also uses `nocheck` by default. Now that those changes are pulled from the Debian package `1.0.1-2` we don't need the changes here, anymore.

Reply:
Oh well, I guess we need to actually drop the `--env=DEB_BUILD_OPTIONS=nocheck `, using the new `autopkgtest` package. Otherwise, the `filter` in `debian/rules` will not match the double `nocheck nocheck` of `DEB_BUILD_OPTIONS`